Sources/PubNubChatComponents/Controllers/ComponentViewController.swift

```diff
@@ -113,6 +113,8 @@ open class CollectionViewComponent: ComponentViewController  {
     configureConstraints()
 
     configureNavigationItem?(navigationItem)
+    
+    configureGestureRecognizers()
```

Comment:
I think I need to know more about code style in this project, don't be angry on my for stupid comments please :) I don't like empty new lines when not necessary (line 114, 116), they are not separating different block of code but similar configuration calls which can be grouped. Additionally line 115 behaves like decorator when other configure methods not.

Reply:
This is what basically the formatter tool should solve. I can adapt to any code style that you guys suggest. Our team gets bigger so we should use the same formatting tools (perhaps with shared settings). Let me know if you think it's worth creating a JIRA issue or discussing it first.